src/XAD/XAD.hpp

```diff
@@ -35,7 +35,9 @@
 #include <XAD/Config.hpp>
 #include <XAD/Exceptions.hpp>
 #include <XAD/Expression.hpp>
+#include <XAD/Hessian.hpp>
 #include <XAD/Interface.hpp>
+#include <XAD/Jacobian.hpp>
```

Comment:
I would not include these 2 headers by default - rather document them and let users include them explicitly if they want this feature.